contributionVerifier.js

```diff
@@ -1,9 +1,20 @@
 const requestp = require('./requestAsPromise');
+const {githubRequest,getReadmeContents} = require('./githubApi');
+
+const clabotToken = process.env.GITHUB_ACCESS_TOKEN;
 
 const contributorArrayVerifier = (contributors) =>
   (committers) =>
     Promise.resolve(committers.filter(c => contributors.indexOf(c) === -1));
 
+const configFileFromGithubUrlVerifier = (contributorListGithubUrl) =>
+  (committers) => githubRequest({
+      url: contributorListGithubUrl,
+      method: 'GET'
+    },clabotToken)
+    .then((body) => githubRequest(getReadmeContents(body),clabotToken))
```

Comment:
the `getReadmeContents` doesn't fetch a README (the name is a bit misleading), it just [fetches the content](https://github.com/ColinEberhardt/cla-bot/blob/master/githubApi.js#L7) of the URL reported by the api.github.com payload (specifically, the `download_url` field).

Based on my local tests, the line is needed, but I may be wrong.